<ui version="4.0" stdsetdef="1" >
  <author>Sebastian Trueg</author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>base_K3bBootImageView</class>
  <widget class="QWidget" name="base_K3bBootImageView" >
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <item>
        <layout class="QHBoxLayout" >
          <item>
            <widget class="QLabel" name="label" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>5</vsizetype>
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text" >
                <string>Boot images:</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="m_buttonNew" >
              <property name="text" >
                <string>&amp;New...</string>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Add new boot image</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="m_buttonDelete" >
              <property name="text" >
                <string>&amp;Delete</string>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Remove selected boot image</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="K3ListView" name="m_viewImages" >
          <property name="allColumnsShowFocus" >
            <bool>true</bool>
          </property>
          <property name="fullWidth" >
            <bool>true</bool>
          </property>
          <column>
            <property name="text" >
              <string>Emulation Type</string>
            </property>
            <property name="clickable" >
              <bool>false</bool>
            </property>
            <property name="resizable" >
              <bool>false</bool>
            </property>
          </column>
          <column>
            <property name="text" >
              <string>Size</string>
            </property>
            <property name="clickable" >
              <bool>true</bool>
            </property>
            <property name="resizable" >
              <bool>true</bool>
            </property>
          </column>
          <column>
            <property name="text" >
              <string>Local Path</string>
            </property>
            <property name="clickable" >
              <bool>false</bool>
            </property>
            <property name="resizable" >
              <bool>false</bool>
            </property>
          </column>
        </widget>
      </item>
      <item>
        <widget class="Q3ButtonGroup" name="m_groupImageType" >
          <property name="title" >
            <string>Emulation Type</string>
          </property>
          <layout class="QHBoxLayout" >
            <item>
              <widget class="QRadioButton" name="m_radioFloppy" >
                <property name="text" >
                  <string>Flopp&amp;y</string>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>Emulate a 1440/2880 kb floppy</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QRadioButton" name="m_radioHarddisk" >
                <property name="text" >
                  <string>Harddisk</string>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>Emulate a harddisk</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QRadioButton" name="m_radioNoEmulation" >
                <property name="text" >
                  <string>None</string>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>No emulation at all</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="Q3GroupBox" name="m_groupOptions" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="title" >
            <string>Settings</string>
          </property>
          <layout class="QVBoxLayout" >
            <item>
              <widget class="QCheckBox" name="m_checkNoBoot" >
                <property name="text" >
                  <string>No boot image</string>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>Do not boot from the emulated floppy/harddisk</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="m_checkInfoTable" >
                <property name="text" >
                  <string>Boot-info-table</string>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QGridLayout" >
                <item row="0" column="1" >
                  <widget class="QLineEdit" name="m_editLoadSegment" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="text" >
                      <string>0</string>
                    </property>
                  </widget>
                </item>
                <item row="1" column="1" >
                  <widget class="QLineEdit" name="m_editLoadSize" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="text" >
                      <string>0</string>
                    </property>
                  </widget>
                </item>
                <item row="0" column="0" >
                  <widget class="QLabel" name="textLabel2_2" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="text" >
                      <string>Boot load segment:</string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="1" column="0" >
                  <widget class="QLabel" name="textLabel3" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="text" >
                      <string>Boot load size:</string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <item>
            <widget class="QPushButton" name="m_buttonToggleOptions" >
              <property name="text" >
                <string>Show Advanced Op&amp;tions</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer16" >
              <property name="sizeHint" >
                <size>
                  <width>261</width>
                  <height>21</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <item>
            <widget class="QLabel" name="textLabel2" >
              <property name="text" >
                <string>Boot catalog:</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="m_editBootCataloge" >
              <property name="text" >
                <string>boot/boot.catalog</string>
              </property>
              <property name="readOnly" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
    <customwidgets>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
      <header location="global" >k3listview.h</header>
    </customwidget>
  </customwidgets>
  <includes>
    <include location="local" >k3listview.h</include>
  </includes>
  <connections>
    <connection>
      <sender>m_radioNoEmulation</sender>
      <signal>toggled(bool)</signal>
      <receiver>textLabel2_2</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
    <connection>
      <sender>m_radioNoEmulation</sender>
      <signal>toggled(bool)</signal>
      <receiver>textLabel3</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
    <connection>
      <sender>m_radioNoEmulation</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_editLoadSegment</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
    <connection>
      <sender>m_radioNoEmulation</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_editLoadSize</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
    <connection>
      <sender>m_radioNoEmulation</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_checkNoBoot</receiver>
      <slot>setDisabled(bool)</slot>
    </connection>
  </connections>
</ui>
